_conferences/flutter-bytes-conf-november-2023.md

```diff
@@ -0,0 +1,14 @@
+---
+name: "Flutter Bytes Conference"
+website: https://www.flutterbytesconf.com/
+location: Lagos, Nigeria
+online: false
+
+date_start: 2023-11-04
+date_end:   2023-11-04
+
+cfp:
+  start: 2023-01-01
```

Comment:
We can just set the start/end date as the day of the conference for now and merge it, later we just fix the dates once available.

Reply:
```suggestion
  start: 2023-11-04
```